jovian/jovian_nb_ext/main.js

```diff
@@ -399,162 +365,153 @@ define([
+          )
+        )
         .append(
           $("<input/>")
-            .addClass("form-check-input")
-            .attr("type", "radio")
-            .attr("name", "cap_opt")
-            .attr("value", "True")
+            .addClass("form-control")
+            .attr("id", "files_box")
+            .attr("placeholder", "utils.py, inputs.csv")
         )
+        .append("<br>");
+
+      const environment = $("<div/>")
+        .append($("<label/>").text("Which type of environment to be captured?"))
         .append(
-          $("<label/>")
-            .addClass("form-check-label")
-            .text("True")
-            .css("margin-right", "1em")
+          $("<select/>")
+            .attr("id", "env_opt")
+            .addClass("form-control")
+            .append($("<option/>").text("auto"))
+            .append($("<option/>").text("conda"))
+            .append($("<option/>").text("pip"))
+            .append($("<option/>").text("None"))
+            .css("margin-left", "0em")
+        )
+        .append("<br>");
+
+      const project_id = $("<div/>")
+        .append(
+          $("<label/>").text(
+            "Name of the Jovian.ml project like `user_name_on_jovian/notebook_name`"
+          )
         )
         .append(
           $("<input/>")
-            .addClass("form-check-input")
-            .attr("type", "radio")
-            .attr("name", "cap_opt")
-            .attr("value", "False")
+            .addClass("form-control")
+            .attr("id", "project_id_box")
         )
+        .append("<br>");
+
+      const new_project = $("<div/>")
+        .append($("<label/>").text("To create a new project?"))
         .append(
-          $("<label/>")
-            .addClass("form-check-label")
-            .text("False")
-        );
+          $("<select/>")
+            .attr("id", "if_new")
+            .addClass("form-control")
+            .append($("<option/>").text("True"))
+            .append($("<option/>").text("False"))
+            .css("margin-left", "0em")
+        )
+        .append("<br>");
 
-      const env_type_label = $("<label/>").text(
-        "Which type of environment to be captured?"
-      );
-      const env_type_box = $("<select/>")
-        .attr("id", "env_opt")
-        .append($("<option/>").text("conda"))
-        .append($("<option/>").text("pip"))
-        .css("margin-left", "1em");
-
-      const notebook_id_label = $("<label/>").text(
-        "Notebook-id(optional) This is picked up by the library automatically. Incase if you want to commit to a different notebook, enter the address of that notebook like `user_name_on_jovian/notebook_name`"
-      );
-      const notebook_id_box = $("<input/>")
-        .addClass("form-control")
-        .attr("id", "notebook_id_box")
-        .attr("placeholder", "None");
+      const privacy = $("<div/>")
+        .append(
+          $("<label/>").text(
+            "Choose the type of notebook when creating a new project"
```

Comment:
Change this to `Notebook privacy settings (applicable while creating a new notebook project)`

Reply:
and remove the functionality of this competent is disabled when `new_project=False`.

**Edge Case**
First time user will have `new_project=False` but still it'll be creating a new project, but he/she can't set this parameter since it will be disabled.

For improvement you can look into to .jovianrc and update these parameters accordingly but lets keep that after launch.